apps/electionguard-cli/ElectionGuard.CLI.csproj

```diff
@@ -21,9 +21,13 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.9.1" />
+    <!-- <PackageReference Include="ElectionGuard.Encryption" Version="1.91.11" /> -->
```

Comment:
had to comment this out and add back the manual library loading because the core changes are not included in nuget (since they are not yet merged in and released). we could change this back in the future but it will continue to be a headache so long as the cli tool is being worked on. maybe we can create separate csproj files or something like that.